cognite/client/_api/geospatial.py

```diff
@@ -160,6 +163,51 @@ def mapper(it):
         res = self._post(url_path=f"{self._RESOURCE_PATH}/featuretypes/update", json=json)
         return FeatureTypeList._load(res.json()["items"], cognite_client=self._cognite_client)
 
+    def patch_feature_types(self, patch: Union[FeatureTypePatch, List[FeatureTypePatch]] = None) -> FeatureTypeList:
+        """`Patch feature types`
+        <https://docs.cognite.com/api/v1/#operation/updateFeatureTypes>
+
+        Args:
+            patch (Union[FeatureTypePatch, List[FeatureTypePatch]]): the patch to apply
+
+        Returns:
+            FeatureTypeList: The patched feature types.
+
+        Examples:
```

Comment:
While you are at it, maybe add an example of multi property index, I think that is not obvious that we can do it.